test/value-binding.html

```diff
@@ -21,28 +21,56 @@
 </head>
 <body>
 
-  <polymer-element name="x-foo" attributes="value" noscript>
+  <dom-module id="x-foo">
```

Comment:
For 0.8 elements, we've begun fixturing DOM using an element called `<test-fixture>`. This part of the documentation is particularly relevant to this use case: https://github.com/Polymer/test-fixture#data-bound-templates

The upside is that you may be able to avoid creating all of those custom elements below just to test behavior of `<core-localstorage>`. Also, the `fixture` helper will automatically clean up the fixtured DOM after each test.